src/cli.js

```diff
@@ -156,3 +169,38 @@ async function upgradeAndDockerize (name) {
   await autoUpgrade();
   dockerize(name);
 }
+
+function updateLastVersionUsed() {
+  const fileHeader = "DO NOT MODIFY";
```

Comment:
Let's strip this header. The reason is, if anyone else wants to read the version from this file, which is why it is being created, they will have to strip it too. It won't be valid json otherwise.